fenix/app/src/main/res/layout/search_widget_large.xml

```diff
@@ -38,6 +38,6 @@
         android:layout_alignParentEnd="true"
         android:layout_centerVertical="true"
         android:layout_marginEnd="1dp"
-        android:background="@drawable/ic_microphone_widget"
+        android:background="@android:color/transparent"
```

Comment:
I wonder if there is a way to add the A-C icon through xml as well?

Reply:
While testing this I noticed that when the device is set to light theme, the icon remains white. The way Focus manages this is by overriding the `mozac_ui_icons_fill` color. Currently we are not doing this in Fenix, however the color is used in many drawables throughout both A-C and Fenix. The alternative to this would be to keep `ic_microphone_widget` as a duplicate of `mozac_ic_microphone` but using `fx_mobile_icon_color_primary` as `fill_color`. What do you think?

Reply:
I like the idea of overriding the `mozac_ui_icons_fill` value! Is there any downside to this? Overriding a color value we can not theme sounds like a good idea, and a better idea than having a duplicate icon just with a different color.

Btw, we could also use tint to override the color. It would require suppressing the warning, but it looks like using `android:tint` could affect negatively only <21 versions, so we should be safe.
```
android:tint="@color/fx_mobile_text_color_primary"
```

Reply:
Thanks for the `android:tint` suggestion! I tried using `app:tint` in a previous iteration but that didn't work since it is part of the `AppCompatImageView`.

Because `mozac_ui_icons_fill` is mostly [used](https://searchfox.org/mozilla-mobile/search?q=mozac_ui_icons_fill&path=&case=false&regexp=false) in the `ui-icons` module, by overriding it, the only risk would be to use `fx_mobile_icon_color_primary` for icons which we may otherwise want to display as white but I'm not sure if there are any such cases in Fenix.

Whether we decide on using `android:tint` or overriding `mozac_ui_icons_fill`, we won't need to update `SearchWidgetProvider`, so I'll dismiss it's changes.

Reply:
I vote for `android:tint` in this case. It is safe – we are targeting 21+. It is localized – it only affects tinting the widget icons, while overriding colors for all icons is riskier at this point. 

If we decide to do that, we should test the change first to see if it doesn't affect the existing coloring. Given the number of icons and existing theming, it looks a bit complex.